Resources/Prototypes/Entities/Objects/Specific/chemical-containers.yml

```diff
@@ -47,6 +47,28 @@
       inHandsFillBaseName: -fill-
     - type: StaticPrice
       price: 60
+    - type: Damageable
+      damageContainer: Inorganic
+    - type: Destructible
+      thresholds:
+      - trigger:
+          !type:DamageTrigger
+          damage: 10
+        behaviors:
+        - !type:PlaySoundBehavior
+          sound:
+            collection: MetalBreak
+            params:
+              volume: -4
+        - !type:SpillBehavior { }
+        - !type:SpawnEntitiesBehavior
+          spawn:
+            SheetPlastic1:
+              min: 0
+              max: 1
```

Comment:
Something we do for nukes is just have a higher damager threshold that just deletes it so entities get cleaned up and it doesn't leave a bunch of debris everywhere.